api/db/populate-notifications.js

```diff
@@ -0,0 +1,222 @@
+const { createPrompt } = require('./utils');
+
+const User = require('../models/User');
+const Interest = require('../models/Interest');
+const Event = require('../models/Event');
+const Attendant = require('../models/Attendant');
+const Comment = require('../models/Comment');
+const Friendship = require('../models/Friendship');
+const Message = require('../models/Message');
+const Notification = require('../models/Notification');
+
+const types = [
+  'confirmEmail',
+  'createEvent',
+  'uploadAvatar',
+  'selectInterests',
+  'newAttendant',
+  'newComment',
+  'newFriendshipRequest',
+  'acceptedFriendshipRequest',
+  'newMessage',
+];
+
+const createNotification = async (notificationParams) => {
+  const notification = new Notification({
+    user: notificationParams.user,
+    type: notificationParams.type,
+    read: notificationParams.read,
+  });
+  if (notification.read) notification.readAt = new Date();
+  if (notificationParams.referencedUser) {
+    notification.referencedUser = notificationParams.referencedUser;
+  }
+  if (notificationParams.referencedEvent) {
+    notification.referencedEvent = notificationParams.referencedEvent;
+  }
+  if (notificationParams.referencedFriendship) {
+    notification.referencedFriendship = notificationParams.referencedFriendship;
+  }
```

Comment:
```suggestion
  const notification = new Notification(notificationParams);
  if (notification.read) notification.readAt = new Date();
```
Somehow I overlooked this simplification in all previous code reviews... :facepalm: